domain/test/validation.test.js

```diff
@@ -15,13 +15,7 @@ describe('validation unit test', function () {
                 critDmg: 60
             };
             let validationResult = validation.checkItem(item);
-            assert(validationResult.item.type === item.type);
-            assert(validationResult.item.minDmg === item.minDmg);
-            assert(validationResult.item.maxDmg === item.maxDmg);
-            assert(validationResult.item.critChance === item.critChance);
-            assert(validationResult.item.critDmg === item.critDmg);
             assert(validationResult.isValid === true);
```

Comment:
Awesome! Tests are much more concise now. And the intentions of what do you actually test for are much more clear. I hope you see the profit too :)